### ISSUE TITLE
Add search to portfolio page

### ISSUE THREAD

**Issue by OpenGlobe** [18F/nsf-sbir#229]
Can we still pass simple search queries while relying on the NSF page for the more advanced searches?

**Comment by OpenGlobe** [comment]
Unblocking.  Let's:  

- Link out to NSF search result pages as designed
- Revisit if we need / want to design call-outs for stats after soft launch.

In the meantime, if we find good homes for the stats elsewhere in the content (on this page or elsehwere), let's do that.

**Comment by femmebot** [comment]
@OpenGlobe we can't just link out to NSF search results pages. the search that I copied and pasted gets directed to a different (google?) page than what is currently on NSF SBIR's portfolio page.

As a test, try searching "Keen Home" All NSF SBIR/STTR Awards on their current search: https://www.nsf.gov/eng/iip/sbir/portfolio.jsp 

Now try the same search on the mockup: https://federalist-proxy.app.cloud.gov/preview/18f/nsf-sbir/explore/portfolio/index.html

Notice where the search results take you?

**Comment by OpenGlobe** [comment]
Hi @kmonterr - is there anyone on your side that would be particularly good to speak to re: search integration?  We have a few technical details we'd like to sort out to try and improve where the queries are sent and how they're being handled.

**Comment by kmonterr** [comment]
hey @OpenGlobe I would start with @billdaus. He will know who best to contact on his team!

**Comment by line47** [comment]
Search is implemented on `dev` and it searches "All NSF SBIR/STTR Awards" - we may want to expand this functionality in the future as a separate issue.
